charms/kfp-ui/requirements.txt

```diff
@@ -0,0 +1,3 @@
+ops==1.1.0
+git+https://github.com/juju-solutions/resource-oci-image@1964d748022b762b9dce6e8bb7bdf12835102c72
```

Comment:
Leaving 1 comment for multiple lines in different files would be a nice GH feature...